lib/WebSocketClient.js

```diff
@@ -112,6 +112,40 @@ function WebSocketClient(config) {
 
 util.inherits(WebSocketClient, EventEmitter);
 
+function handleWsfProtocol(url) {
+  var path = require('path');
+  var fs = require('fs');
```

Comment:
no problem, let's move it out

meno

On Thu, Jun 16, 2016, 23:26 Iñaki Baz Castillo <notifications@github.com>
wrote:

> In lib/WebSocketClient.js
> <https://github.com/theturtle32/WebSocket-Node/pull/240#discussion_r67427092>
> :
>
> > @@ -112,6 +112,40 @@ function WebSocketClient(config) {
> >
> >  util.inherits(WebSocketClient, EventEmitter);
> >
> > +function handleWsfProtocol(url) {
> > +  var path = require('path');
> > +  var fs = require('fs');
>
> I don't like at all this. Modules should not be loaded within functions
> but on the top of the file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/theturtle32/WebSocket-Node/pull/240/files/91c9b1d2ab8bb696192a7ee7b63bcafb982ef665#r67427092>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABLJvfQ3PNafTCAYIORlUE8TvYZUxzyks5qMb-GgaJpZM4I3T8z>
> .
>